src/m_misc.c

```diff
@@ -2581,10 +2581,10 @@ default_t defaults[] = {
   //
 
   { // no color changes on status bar
-    "sts_always_red",
-    (config_t *) &sts_always_red, NULL,
-    {1}, {0,1}, number, ss_stat, wad_yes,
-    "1 to disable use of color on status bar"
+    "sts_colored_numbers",
```

Comment:
Kicking peoples config again? 😉

Reply:
I know, it's annoying, but so is the reversed order of the original variable. Should I revert the change?

Reply:
No. I agree that the former double-negated variable was a PITA.

Reply:
Ah, sounds good 🙂

I made this PR because I've already felt like it was missing several times while changing things in this menu. That, plus the request here: https://github.com/fabiangreffrath/woof/issues/1457 and the user on Jimmy's Discord, makes me think we should add it back before the next release.

Reply:
Ah, sounds good 🙂

I made this PR because I've already felt like it was missing several times while changing things in this menu. That, plus the request here: https://github.com/fabiangreffrath/woof/issues/1457 and the user on Jimmy's Discord, makes me think we should add it back before the next release.

Reply:
Ah, sounds good 🙂

I made this PR because I've already felt like it was missing several times while changing things in this menu. That, plus the request here: https://github.com/fabiangreffrath/woof/issues/1457 and the user on Jimmy's Discord, makes me think we should add it back before the next release.